shared/cliconfig/file.go

```diff
@@ -11,17 +13,57 @@ import (
 	"github.com/lxc/incus/v6/shared/util"
 )
 
+func getConfigPaths() (string, string, error) {
+	// Figure out the config directory and config path
+	var configDir string
+	if os.Getenv("INCUS_CONF") != "" {
+		configDir = os.Getenv("INCUS_CONF")
+	} else if os.Getenv("HOME") != "" && util.PathExists(os.Getenv("HOME")) {
+		configDir = path.Join(os.Getenv("HOME"), ".config", "incus")
+	} else {
+		usr, err := user.Current()
+		if err != nil {
+			return "", "", err
+		}
+
+		if util.PathExists(usr.HomeDir) {
+			configDir = path.Join(usr.HomeDir, ".config", "incus")
+		}
+	}
+
+	if configDir == "" {
+		return "", "", nil
+	}
+
+	configPath := os.ExpandEnv(path.Join(configDir, "config.yml"))
+
+	return configPath, filepath.Dir(configPath), nil
+}
+
 // LoadConfig reads the configuration from the config path; if the path does
 // not exist, it returns a default configuration.
```

Comment:
That description needs updating as it no longer matches what the function does.

Reply:
Something like:
``` go
// LoadConfig reads the configuration from the config path; if the path does
// not exist, it returns a default configuration; if the given path is empty
// it tries to determine automatically the configuration file to load.
```

Reply:
Any advice?

Reply:
Sorry, been traveling this week and struggling to keep up with Github :)
Yeah, that comment looks good to me.

Reply:
Done!